package.json

```diff
@@ -25,10 +25,10 @@
     "storybook": "start-storybook -p 9001 -c .storybook",
     "test": "npm run test:lint && npm run test:unit && npm run dataValidate",
     "test:accessibility": "npm run killApp && npm run build && run-p --race start bbcA11y",
-    "test:accessibility:ci": "npm run build && run-p --race start bbcA11y",
+    "test:accessibility:ci": "npm install bbc-a11y && run-p --race start bbcA11y",
```

Comment:
The scripts `npm run test:accessibility:ci` and `npm run test:e2e:ci` won't work locally like this, and their only use is on Travis, so I suggest replacing these with a single script that runs this:

`"npm run build && npm install cypress bbc-a11y && run-p --race start cypress && run-p --race start bbcA11y",`

Then in the Travis config, there will just be a single script run for the production test.

Reply:
This is fine BUT the npm run build should be distinct as it's actually testing that the production build can build it isn't really a part of the remainder of the tests, though it is a prerequisite.